### PR TITLE
Payment backend

### PR DESCRIPTION
**Pull request by MangeshThakre**

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -10,11 +10,11 @@ const cors = require("cors");
...
 
 // routes
-const authRouter = require("./router/authRouter.js");
-const paymentRouter = require("./router/paymentRouter.js");
-const contentRoute = require("./router/contentRouter");
-const userRouter = require("./router/userRouter.js");
-const miscRoute = require("./router/miscRouter.js");
+const authRouter = require("./router/auth.router.js");
+const paymentRouter = require("./router/payment.router.js");
+const contentRoute = require("./router/content.router");
```
Comment (shivamvijaywargi) [inline backend/app.js:15]: Add .js here

File: backend/controller/auth.controller.js [hunk 2]
```diff
@@ -300,7 +299,8 @@ const signOut = asyncHandler(async (req, res, next) => {
     httpOnly: true,
     maxAge: new Date().now, //  current date
     path: "/",
-    sameSite: "Lax",
+    sameSite: process.env.NODE_ENV === "production" ? "None" : "Lax",
+    secure: process.env.NODE_ENV === "production" ? true : false,
```
Comment (shivamvijaywargi) [inline backend/controller/auth.controller.js:304]: Have you tested this locally? Not in postman but in browser as well?

File: backend/router/misc.router.js [hunk 3]
```diff
@@ -5,8 +5,7 @@ const {
   getUsersStatistics,
   getMoviesStatistics,
   getSeriesStatistics,
-  getPaymentStatistics,
-} = require("../controller/miscController.js");
+} = require("../controller/misc.controller.js");
 const authorizeRoles = require("../middleware/authorizeRoles");
```
Comment (shivamvijaywargi) [inline backend/router/misc.router.js]: .js, please check all of these, either add them everywhere or remove them from everywhere.

**Review by shivamvijaywargi — CHANGES_REQUESTED** [review]
Please check the comments

File: backend/app.js (hunk 1 above)
Comment (MangeshThakre) [reply backend/app.js:15]: added

File: backend/controller/auth.controller.js (hunk 2 above)
Comment (MangeshThakre) [reply backend/controller/auth.controller.js:304]: tested

File: backend/router/misc.router.js (hunk 3 above)
Comment (MangeshThakre) [reply backend/router/misc.router.js]: added .js

**Review by shivamvijaywargi — APPROVED** [review]

**Pull request merged**